lib/utils.js

```diff
@@ -35,17 +36,26 @@ exports.basicAuth = function(username, password) {
 
 exports.findAvailablePort = function(app){
 
-  var port = (process.env.PORT || 3000);
+  var port = null;
+
+  try {
+    port = Number(fs.readFileSync(__dirname+'/../port.tmp'));
```

Comment:
.tmp is a naming convention - a temporary file that can be deleted with no consequences. I don't think adding a prefix . will add much value to that?